app/controllers/api/v4/converts_controller.rb

```diff
@@ -36,10 +36,14 @@ def create
       new_extension = params[:format]
     end
     filename_without_extension = params[:file].original_filename.split(old_extension)[0]
+    if filename_without_extension.present? && filename_without_extension[-1] == '.'
+      filename_without_extension = filename_without_extension.chop
+    end
```

Comment:
are double periods a problem?

Reply:
When I was testing yes.  It splits on the filetype leaving the period.  But sometimes the filename doesn't exist.

Reply:
Nah this is fine, was just curious what the need was